tests/e2e_regression/edenfast/e2e_config_1.yml

```diff
@@ -0,0 +1,57 @@
+# Config for phase 1 of 2 of e2e regression tests.
```

Comment:
Can you add a paragraph describing the goal of this suite. Basically its relation to `eden`.

Reply:
adding